docs.html

```diff
@@ -98,7 +98,80 @@ <h2>Blog Title</h2>
+                    style="width: 80%; height: 316px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+              </div>
+          </div>
+          <div class="display-code-content">
+            <h2>Outline Buttons</h2>
+            <div class="btn-container">
+                <button class="btn btn-primary-outline">Click Here</button>
+                <button class="btn btn-second-outline">Click Here</button>
+                <button class="btn btn-success-outline">Click Here</button>
+                <button class="btn btn-danger-outline">Click Here</button>
+                <button class="btn btn-warning-outline">Click Here</button>
+                <button class="btn btn-dark-outline">Click Here</button>
+            </div>
+            <div>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C184%2C195%2C0%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520btn-primary-outline%2522%253EClick%2520Here%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-second-outline%2522%253EClick%2520Here%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-success-outline%2522%253EClick%2520Here%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-danger-outline%2522%253EClick%2520Here%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-warning-outline%2522%253EClick%2520Here%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-dark-outline%2522%253EClick%2520Here%253C%252Fbutton%253E"
+                    style="width: 80%; height: 298px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+          </div>
+          <div class="display-code-content">
+              <h2>Buttons with Icons</h2>
+              <div class="btn-container">
+                  <button class="btn btn-primary btn-icon">
+                    <svg class="icon" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M10 18V16H8V14H10V12H12V14H14V16H12V18H10Z" fill="currentColor" /><path fill-rule="evenodd" clip-rule="evenodd" d="M6 2C4.34315 2 3 3.34315 3 5V19C3 20.6569 4.34315 22 6 22H18C19.6569 22 21 20.6569 21 19V9C21 5.13401 17.866 2 14 2H6ZM6 4H13V9H19V19C19 19.5523 18.5523 20 18 20H6C5.44772 20 5 19.5523 5 19V5C5 4.44772 5.44772 4 6 4ZM15 4.10002C16.6113 4.4271 17.9413 5.52906 18.584 7H15V4.10002Z" fill="currentColor" /></svg>
```

Comment:
Just a suggestion: Use Font-awesome Icons here.

Reply:
I am more use to using SVG from next I will use Font Awesome Icons

Reply:
Alright. Then its fine